### PR TITLE
fixes bookmark thumb and summary display

### PR DESCRIPTION
**Pull request by esotrope**
** github not able to upload images at the moment so i'm dropping them on your doorstep in slack **

### CODE REVIEW

File: webapp/src/app/resource/components/resource-properties/resource-properties.component.html [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <h2 class="offscreen">Resource Properties</h2>
-<div *ngIf="!svgSafeUrl" class="image thumbnail">
-  <img [src]="properties.image | replaceNullImage" aria-hidden="true">
-</div>
-<div *ngIf="svgSafeUrl" class="image {{ subjectCode }}">
-  <object type="image/svg+xml" [data]="svgSafeUrl" aria-hidden="true"></object>
+<div class="thumbnail-container">
```
Comment (esotrope) [inline webapp/src/app/resource/components/resource-properties/resource-properties.component.html]: some component reuse yay

File: webapp/src/app/resource/components/resource-properties/resource-properties.component.scss [hunk 2]
```diff
@@ -1,47 +1,22 @@
 @import "colors";
 @import "typography";
 @import "spacing";
-@import "@material/ripple/mdc-ripple";
 
 :host {
```
Comment (esotrope) [inline webapp/src/app/resource/components/resource-properties/resource-properties.component.scss]: simplified styles here

File: webapp/src/app/resource/components/resource-properties/resource-properties.component.ts [hunk 3]
```diff
@@ -1,31 +1,15 @@
...
+  styleUrls: ['./resource-properties.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class ResourcePropertiesComponent implements OnInit {
+export class ResourcePropertiesComponent {
 
   @Input()
   properties: ResourceProperties;
 
-  svgSafeUrl: SafeResourceUrl;
```
Comment (esotrope) [inline webapp/src/app/resource/components/resource-properties/resource-properties.component.ts]: this logic is now in the shared comp

File: webapp/src/app/resource/components/resource-summary-card/resource-summary-card.component.html [hunk 4]
```diff
@@ -1,32 +1,38 @@
...
-      </span>
-    </div>
-
-    <div *ngSwitchCase="resourceTypes.ProfessionalLearning" class="details professional-learning caption emphasis-medium">
-      <dynamic-html [content]="resourceSummary.summary"></dynamic-html>
-    </div>
-
-    <div *ngSwitchCase="resourceTypes.AccessibilityStrategy" class="details professional-learning caption emphasis-medium">
-      <dynamic-html [content]="resourceSummary.summary"></dynamic-html>
-    </div>
```
Comment (esotrope) [inline webapp/src/app/resource/components/resource-summary-card/resource-summary-card.component.html]: these summaries are what went away

Comment (jdbernard) [reply webapp/src/app/resource/components/resource-summary-card/resource-summary-card.component.html]: We need to double-check that we want these summaries to go away. I think I recall them asking for different things depending on the resource type:

* Instructional and Connections Playlist: summary details (grade, target, claim, etc.)
* Prof. Learning and Strategies: the summary content

Comment (esotrope) [reply webapp/src/app/resource/components/resource-summary-card/resource-summary-card.component.html]: I'll ask in the parking lot 👍

**Comment by esotrope** [comment]
Does the svg padding/border apply in the properties view?
Or maybe it is better asked - should the svg in the property view have padding but the normal images not?

**Comment by jdbernard** [comment]
Yes, the SVGs need padding in resource properties as well. Regular images should go all the way to the border (and be trimmed slightly by the rounded corners).

**Pull request merged**